site/get-started.markdown

```diff
@@ -0,0 +1,143 @@
+> stack --version                               # If you installed Stack
+```
+
+### Editor
+HLS (Haskell Language Server), which you just installed via GHCup, is the one that brings all the cool IDE features to editors, so as long as your editor has a decent Haskell language extension that utilizes HLS, you are good.
+
+The most straightforward option is to go with **Visual Studio Code** - just install [Haskell extension](https://marketplace.visualstudio.com/items?itemName=haskell.haskell) and you are all set, it should work out of the box and use your installation of HLS.
+
+Of other editors that have good Haskell extensions, the most popular ones are Vim and Emacs.
+
+## Running first lines of code
+
+We have everything set up, let's use it!
+
+`GHC` brings an interactive interpreter called `GHCi` together with it, which is great for playing with Haskell and trying things out, so let's give it a spin.
+So even though Haskell is a compiled language, it comes with an interpreter too -> how cool is that?
+
+Run `ghci`, which should start a new prompt for you.
+
+Let's do a simple calculation to check Haskell's computing capabilities:
+```
+> 6 + 3^2 * 4
+42
+```
+Hey, they call Haskell lazy, but that was quick!
+
+42 is a nice even number, but what about the even numbers after it? Let's get the first 10 even numbers after 42.
+```
+> take 10 $$ filter even [43..]
+[44,46,48,50,52,54,56,58,60,62]
+```
+
+What is the sum of those?
+```
+> sum it
+530
+```
+**NOTE**: We used a special feature of GHCi here, which is a special `it` variable that remembers the result of the last expression.
+
+Great, you got the first taste of Haskell! Now let's get to running a real program.
+
+## Writing your first Haskell program
+
+In your editor, create a new file named `hievb.hs`.
+
+Write the following in it:
+```
+main = do
+  putStrLn "Hi, everybody!"
+  putStrLn $$ "Pls look at my favorite odd numbers: " ++ filter odd [10..20]
+```
+
+You can now compile it with `ghc`, which will produce `hievb` binary that we will then run:
+```
+> ghc hievb.hs
+> ./hievb
+Hi, everybody!
+Pls look at my favorite odd numbers: [11,13,15,17,19]
+```
+
+There you go, you just wrote a short, polite program in Haskell!
+
+**TIP**: To interpret the source file directly, without producing any build artifacts, you can use the special `runghc` command like this:
+```
+> runghc hievb.hs
+Hi, everybody!
+Pls look at my favorite odd numbers: [11,13,15,17,19]
+```
+
+**GHCI TIP**: You can also load your file directly into `ghci`, which will enable you to play with any functions and other definitions you defined in it. So for our example, we can just load hievb.hs with ghci and then call the function `main` like this:
+```
+> ghci hievb.hs
+... Compiling ...
+> main
+Hi evb!
+```
+
+**PRO TIP**: Once you progress a bit more in your Haskell journey, you will very likely want to use `cabal` or `Stack` for building your projects, and not `ghc` directly.
```

Comment:
Unless you are showing how, it is better to remove this. Their Haskell tutorial of choice should explain that.

Reply:
Why?

Reply:
Because imo it's not fun to be told "I'm showing you how to do this, but actually you want to do it in a different way" without actually explaining what that way is. I'm just left scratching my head.

Reply:
Ok, removed!